README.md

```diff
@@ -1,15 +1,37 @@
-# OpenTelemetry
-[![Gitter chat][gitter-image]][gitter-url]
-![Continuous Build](https://github.com/open-telemetry/opentelemetry-java/workflows/Continuous%20Build/badge.svg)
-[![Coverage Status][codecov-image]][codecov-url]
-[![Maven Central][maven-image]][maven-url]
-
-We hold regular meetings. See details at [community page](https://github.com/open-telemetry/community#java-sdk).
-
-## Overview
-
-OpenTelemetry is the merging of OpenCensus and OpenTracing into one
-project.
+<p align="center">
+  <strong>
+    <a href="https://opentelemetry.io/docs/java/getting_started/#automatic-instrumentation">Getting Started<a/>
+    &nbsp;&nbsp;&bull;&nbsp;&nbsp;
+    <a href="https://github.com/open-telemetry/community#special-interest-groups">Getting Involved<a/>
+    &nbsp;&nbsp;&bull;&nbsp;&nbsp;
+    <a href="https://gitter.im/open-telemetry/opentelemetry-java">Getting In Touch<a/>
+  </strong>
+</p>
+
+<p align="center">
+  <a href="https://circleci.com/gh/open-telemetry/opentelemetry-java">
+    <img alt="Build Status" src="https://img.shields.io/circleci/build/github/open-telemetry/opentelemetry-java?style=for-the-badge">
```

Comment:
we no longer build on circleci. See the build link to GHAs above, which got removed.